schematron/v2.1.0/tests/test_L000_Prelim_Analysis.py

```diff
@@ -0,0 +1,85 @@
+import os
```

Comment:
Alright - so per 211 5.2.3: "Summary information about bench- marking energy use, based on 2003 CBECS data, is presented in the 2015 ASHRAE Handbook—HVAC Applications, Chap- ter 36, “Energy Use and Management.” More recent 2012 CBECS data are published by the DOE Energy Information Administration. Additional data are available from ENERGY STAR Portfolio Manager, which compares measured energy data to that of a peer building (by building type and weather). Data specific to multifamily housing are available from 2009 RECS statistics or from EPMI."

My takeaway from this is that we should probably making some checks for the building based on metadata used to compare the 'measured' building to the 'benchmark' data set. I think:
- Required:
    - `auc:BuildingClassification`
    - `auc:OccupancyClassification`
    - `auc:YearOfConstruction`
- Recommended:
    - One of: `auc:ClimateZoneType/*/auc:ClimateZone` or `auc:Address` with the city and state
    - Gross floor area

Those to me become minimum requirements for summary information necessary to accurately benchmark your building.

Reply:
and I think we should also write tests for those as well.

Reply:
updated 👍